ci-operator/step-registry/cri-o/e2e/test/cri-o-e2e-test-commands.sh

```diff
@@ -58,6 +59,5 @@ timeout --kill-after 10m 400m gcloud compute ssh --zone="${ZONE}" ${instance_nam
     tar -xzvf cri-o.tar.gz -C "\${REPO_DIR}"
     cd "\${REPO_DIR}/contrib/test/ci"
     echo "localhost" >> hosts
-    ansible-playbook e2e-main.yml -i hosts -e "TEST_AGENT=prow" --connection=local -vvv --tags setup,e2e
+    ansible-playbook e2e-main.yml -i hosts -e "TEST_AGENT=prow USE_CONMONRS=$USE_CONMONRS" --connection=local -vvv --tags setup,e2e
```

Comment:
in the other variants, we use an ansible variable, not a env variable. Part of me thinks they should be consistent (the other part doesn't really care :) )

Reply:
I'm wondering how it would fit into the current structure. Would that mean that we have to create a copy of the whole workflow to avoid the nev variable? (My goal was to inject as less code as possible since we can switch to conmonrs at a certain point in time)

Reply:
@wgahnagl @sohankunkerkar do y'all have any preferences

Reply:
> I think you can default the variable to false in cri-o's main.yml, then run certain installation steps if the variable `use_conmonrs` is true

Yeah, this sounds good. We can also use `--extra-vars` to pass that variable

Reply:
Yeah, looks reasonable to me.